keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,266 @@
+* Review At Risk KEPS for my SIG
+    * Help Prioritize last minute reviews
+* Review Exceptions for my SIG
+    * Determine release blocking / major importance
+* Review Slipped Enhancements to support planning
+    * Determine what phase it missed
+    * Identify multiple slips / misses
+
+#### Enhancements Subteam
+
+* Pre enhancements Freeze (Enhancements Collection)
+  * Collect intents of KEP authors to graduate a KEP
+  * Ensure that those intents comply to the graduation requirements
+  * Mark enhancements "At Risk" based on compliance with graduation requirements
+
+* At Enhancements Freeze
+  * Remove enhancements that do not comply with requirements
+
+* Enhancements Freeze - Code Freeze
+  * Ensure that the k/k PRs for the KEP are populated
+  * Mark enhancements "At Risk" based on progress on implementation
+  * Approve/Deny exceptions for Enhancements Freeze
+
+* Around Code Freeze
+  * Ensure that the enlisted k/k are merged
+  * Approve/Deny exceptions for Code Freeze
+
+* At Code Freeze
+  * Remove enhancements that have not completed requirements for Code Freeze
+
+* Throughout the Cycle
+  * Prepare a manifest of what's the status of enhancements at any point of time
+
+#### Docs Subteam
+
+* Enhancements Freeze - Open Docs PR Deadline
+  * Ensure that the k/w PRs are populated in the KEPs
+  * Communicate to SIG Docs about the expected review load
+
+* Open Docs PR Deadline - Docs Complete
+  * Mark enhancements "At Risk" if the following are true:
+    *  Docs PRs are not updated
+    *  The draft PRs have not been updated
+
+### Notes/Constraints/Caveats
+
+This KEP is based on a [high level plan](https://docs.google.com/document/d/1qnfXjQCBrikbbu9F38hdzWEo5ZDBd57Cqi0wG0V6aGc/edit#) incorporating feedback collected from release team members and contribtors over several cycles.
+
+In 1.21, we'll focus on implementing much of this process manually so there will be some burdon placed on people to create the appropriate YAML structure, but ideally these changes will free up time for people on the release team to work on automating
+
+### Risks and Mitigations
+
+* Conflicts can arise when two people update/modify the same artifact
+  * The team needs to have Git experience
+  * Only one person works on the artifacts at a time
+  * Communicate with each other when making changes
+  * Bots can eventually handle manifest generation
+* Keeping KEP implementation history updated
+  * Requires intervention from an human and trust needs to be on the same person
+* Preventing enhancements to slip across release cycles
+* Ensuring enhancements don't permanently remain in Beta stage
+* This KEP doesn't talk about KEP owners keeping enhancements issues up-to-date
+
+## Design Details
+
+TODO: - Write the folder structure and the format of receipts.
+
+### Test Plan
+
+This KEP entails a process change and hence the tooling to ease the processes will have necessary tests.
+
+### Graduation Criteria
+
+#### Alpha
```

Comment:
If this is data as input for new KEPs as a guideline, then I would add e2e tests cover the happy path